### ISSUE TITLE
raise ValueError(data["status"])  ValueError: REQUEST_DENIED

### ISSUE THREAD

**Issue by deepeshguru** [datadesk/python-googlegeocoder#20]
from googlegeocoder import GoogleGeocoder
geocoder = GoogleGeocoder()
search = geocoder.get("Watts Towers")
Traceback (most recent call last):

  File "<ipython-input-88-b7999bb99344>", line 3, in <module>
    search = geocoder.get("Watts Towers")

  File "C:\Users\Raushan\Anaconda3n\lib\site-packages\googlegeocoder\__init__.py", line 50, in get
    raise ValueError(data["status"])

ValueError: REQUEST_DENIED

**Comment by pikaachu** [comment]
**from googlegeocoder import GoogleGeocoder**
**geocoder = GoogleGeocoder()**
**search = geocoder.get("Watts Towers")**

>Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
      File "/usr/local/lib/python2.7/dist-packages/googlegeocoder/__init__.py", line 50, in get
        raise ValueError(data["status"])
      ValueError: REQUEST_DENIED

**Comment by palewire** [comment]
I suspect that Google is now enforcing a requirement for an API key. If that's right, this library would need to be updated.

**Comment by palewire** [comment]
I can confirm that I am getting the same error.

```python
>>> from googlegeocoder import GoogleGeocoder
>>> geocoder = GoogleGeocoder()
>>> search = geocoder.get("Watts Towers")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/palewire/Code/python-googlegeocoder/googlegeocoder/__init__.py", line 50, in get
    raise ValueError(data["status"])
ValueError: REQUEST_DENIED
```

**Comment by palewire** [comment]
Google now requiresd an API key for this library to work.

I have added support providing your credentials to the client when it is initialized, as well as tests and documentation to reflect the change.

A new version of the library has been released on PyPI You'll want to run the following:

```bash
pip install python-googlegeocoder --upgrade
```

And update your Python code to something like:

```python
>>> from googlegeocoder import GoogleGeocoder
>>> geocoder = GoogleGeocoder("<YOUR GOOGLE MAPS API KEY>")
>>> search = geocoder.get("Watts Towers")
```

I know it's hassle for users, but it's beyond my control.
